SolidityX/package.json

```diff
@@ -7,6 +7,7 @@
     "quais-polling": "^1.0.4"
   },
   "devDependencies": {
-    "@nomicfoundation/hardhat-toolbox": "^4.0.0"
+    "@nomicfoundation/hardhat-toolbox": "^4.0.0",
+    "@nomicfoundation/hardhat-verify": "^2.0.4"
```

Comment:
Do we need to install the `hardhat-verify` plugin? I'm pretty sure it's wrapped into the already installed `hardhat-toolbox` as you can see [here](https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-toolbox).